### PR TITLE
Routes update

### PR DESCRIPTION
**Pull request by Guite**
| Q                 | A
| ----------------- | ---
| Bug fix?          | yes
| New feature?      | no
| BC breaks?        | no
| Deprecations?     | no
| Fixed tickets     | -
| Refs tickets      | -
| License           | MIT
| Changelog updated | no

### CODE REVIEW

File: src/system/RoutesModule/Form/Type/Base/AbstractRouteType.php [hunk 1]
```diff
@@ -82,7 +81,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
      */
     public function addEntityFields(FormBuilderInterface $builder, array $options)
     {
-
+
```
Comment (craigh) [inline src/system/RoutesModule/Form/Type/Base/AbstractRouteType.php]: ick

File: src/system/RoutesModule/Form/Type/Base/AbstractRouteType.php [hunk 2]
```diff
@@ -173,10 +167,9 @@ public function addEntityFields(FormBuilderInterface $builder, array $options)
                 'class' => '',
                 'title' => $this->__('Enter the host of the route')
             ],
-            'required' => false
-            ,
+            'required' => false,
         ]);
-
+        
         $listEntries = $this->listHelper->getEntries('route', 'schemes');
```
Comment (craigh) [inline src/system/RoutesModule/Form/Type/Base/AbstractRouteType.php]: any way to eliminate these useless spaces/tabs from the generator? overtime my IDE opens these, it corrects them and then the generator switches them back.

Comment (Guite) [inline src/system/RoutesModule/Form/Type/Base/AbstractRouteType.php]: vendor issue https://bugs.eclipse.org/bugs/show_bug.cgi?id=398172

File: src/system/RoutesModule/Form/Type/Base/AbstractRouteType.php (hunk 1 above)
Comment (craigh) [inline src/system/RoutesModule/Form/Type/Base/AbstractRouteType.php]: This isn't a blank space problem. this empty should not be here at all.

Comment (Guite) [inline src/system/RoutesModule/Form/Type/Base/AbstractRouteType.php]: Fixed in https://github.com/Guite/MostGenerator/commit/acefde8dab75baa4264fabf5244f8f77a38f1298

Comment (craigh) [inline src/system/RoutesModule/Form/Type/Base/AbstractRouteType.php]: 👍 ✅ you can delete this branch too I think

Comment (Guite) [inline src/system/RoutesModule/Form/Type/Base/AbstractRouteType.php]: It will be recreated by the CI Job anyway... nvm

**Pull request merged**